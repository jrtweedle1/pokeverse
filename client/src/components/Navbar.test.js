import React from "react";
import { unmountComponentAtNode } from "react-dom";

