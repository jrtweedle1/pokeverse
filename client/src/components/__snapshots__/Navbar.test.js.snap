// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the component renders correctly 1`] = `
<nav
  className="navbar navbar-expand-lg bg-body-tertiary"
>
  <div
    className="container-fluid"
  >
    <img
      alt="Pokeverse logo in "
      src="pokeverse.png"
    />
    <button
      aria-controls="navbarNav"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-bs-target="#navbarNav"
      data-bs-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarNav"
    >
      <ul
        className="navbar-nav"
      >
        <li
          className="nav-item"
        >
          <a
            aria-current="page"
            className="nav-link active"
            href="/"
          >
            Home
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
